MiniSpotify/MiniSpotify/Source/APIRequestor.cs

```diff
@@ -209,15 +212,31 @@ public void ModifyPlayback()
             }
         }
 
-        public bool SkipSongPlayback(bool a_nextSong = true)
+public bool SkipSongPlayback(bool a_nextSong = true)
         {
             //See if they're already listening to music.
             if (m_spotifyWebAPI.GetPlayingTrack().IsPlaying)
             {
                 if (a_nextSong)
-                    m_spotifyWebAPI.SkipPlaybackToNext();
+                { try
+                    {
+                        m_spotifyWebAPI.SkipPlaybackToNext();
+                    }catch(ArgumentException e) // Spammed probaly the button
+                    {
+                        Console.WriteLine(e.StackTrace);
+                    }
+                }
                 else
-                    m_spotifyWebAPI.SkipPlaybackToPrevious();
+                {
+                    try
+                    {
+                        m_spotifyWebAPI.SkipPlaybackToPrevious();
+                    }
+                    catch (ArgumentException e) // Spammed probaly the button
+                    {
+                        Console.WriteLine(e.StackTrace);
+                    }
+                }
```

Comment:
I'm happy with this for now, it's better than what I was doing.

I'll modify this later, I think I want to add some kind of log functionality so we can have crash logs.